NAMESPACE

```diff
@@ -7,6 +7,7 @@ export(.showScalarArguments)
 export(AffinityParam)
 export(AgnesParam)
 export(ClaraParam)
+export(DMMParam)
```

Comment:
In general, new classes should follow Pascal case, even when the "word" is an acronym, e.g., `SomParam`. The exceptions are some of the NN-related params which still use all-caps acronyms for back-compatibility.